### DISCUSSION TITLE
Fehler bei der Erkennung

### DISCUSSION THREAD

**Discussion by RasputinBerry** [jomjol/AI-on-the-edge-device#1708] [General] [answered]
Hallo, ein Bild sagt mehr als tausend Worte ...
Hier müßte doch 182.8088 stehen, oder ? Kann man etwas an den Einstellungen ändern / kontrolliieren ?
Danke für Hinweise.

![Bild](https://user-images.githubusercontent.com/56359831/209777094-0c7f4a77-247e-4c31-8db4-d06f73613e87.jpg)

**Comment by ghost** [comment]
Kannst du mal die Recognition dazu posten?

Und deine Firmware-Version?

**Comment by ghost** [comment]
Wenn deine Version >=13.0 ist, kannst du "AnalogDigitalTransitionStart" mal auf 7.5 stellen. Aber nur, wenn das Digit wirklich schon bei 7.5 anfängt zu wechseln. Ansonsten ist 183 nämlich richtig.

**Comment by RasputinBerry** [comment]
... habe gerade aufv13.0.8 upgedatet. Probiere jetzt die "7.5" aus. Danke für die Hilfe.

![grafik](https://user-images.githubusercontent.com/56359831/209806150-ebd53e0f-d58c-4a96-88fe-b77d22397847.png)

![grafik](https://user-images.githubusercontent.com/56359831/209806591-918adb06-adc1-431e-bbc1-f741f2c3e3cd.png)

**Comment by ghost** [comment]
Ich glaube bei dein Zähler ist ähnlich diesem https://github.com/jomjol/AI-on-the-edge-device/discussions/1697#discussion-4696486 .  Das letzte Digit läuft sehr früh los.
Es gibt andere bei denen "hängt" das letzte Digit bei 2.8 bis der analoge Zeiger >9 ist.

**Comment by RasputinBerry** [comment]
Aha ... also sollte ich beim nächsten Wechsel der "3" auf die "4" beobachten wann der Wechsel startet und ggf. deinen Vorschlag "7.5" anpassen, tendenziell liegt der früher als "7.5".

Danke soweit.

**Comment by ghost** [comment, marked as answer]
Da die Frage häufiger kommt, habe ich mal ein Wiki zum Verständnis gemacht.

https://github.com/jomjol/AI-on-the-edge-device/wiki/Watermeter-specific-analog---digital-transition

**Comment by RasputinBerry** [comment]
Super, vielen Dank.

**Comment by RasputinBerry** [comment]
Ganz andere (peinliche) Frage: 
Die Antwort zum ursprünglichen Problem ist ja da ...
Wie verschwindet jetzt 
![grafik](https://user-images.githubusercontent.com/56359831/209811182-6af70e42-c563-4e42-ac13-601d5d557933.png)
und
![grafik](https://user-images.githubusercontent.com/56359831/209811024-36ee52b6-8e9a-490f-b70b-f32692d766f0.png)

**Reply by ghost** [reply]
<img width="407" alt="image" src="https://user-images.githubusercontent.com/412645/209814717-bc1cea31-6050-413e-ae11-1c6b7e4a7a0f.png">

**Comment by RasputinBerry** [comment]
... doch noch eine Verständnisfrage ... z.Zt. sieht der Zähler so aus:

![grafik](https://user-images.githubusercontent.com/56359831/210005267-9b4bdb1a-7a2e-431a-80f8-5449f4d90932.png)

Die "3" wechselt also langsam ... auf was bzw. welches Elemant bezieht sich der "AnalogDigitalTransitionStart" Wert ?

**Reply by ghost** [reply]
AnalogDigitalTransitionStart kann man minimal 6 einstellen. Das sollte auch so passen. 
Hauptsächlich der Bereich zwischen 8 und 2 des analogen Zeigers ist etwas anders, da leider analoger Zeiger und letztes Digit nicht immer so schön wie hier parallel laufen.

**Reply by RasputinBerry** [reply]
Sorry, ich steh schon wieder / immer noch auf dem Schlauch ...
Du schreibst: "_... da leider analoger Zeiger und letztes Digit nicht immer so schön wie hier parallel laufen._".
Muss ich das so verstehen, dass ich (eigentlich) keine Änderung des "AnalogDigitalTransitionStart" brauche ?! Oder bei welcher Stellung des analogen Zeigers muss oder sollte ich den Wert (Wie ?, siehe erstes Bild ganz oben ) einstellen 

![grafik](https://user-images.githubusercontent.com/56359831/210103963-78048b51-c920-4ad9-afca-7d5e39db0078.png)

**Reply by ghost** [reply]
Nein. Leider nicht. Nur weil sie schön parallel laufen, wirst du ab 3.8 hier Probleme haben. Deshalb AnalogDigitalTransitionStart besser auf 7 stellen, sonst wird ab x.8 des digits gerundet, was zu falschen Ergebnissen führt.

**Reply by RasputinBerry** [reply]
Danke ... ich beobachte weiter ...
